addons/hr_holidays/static/tests/helpers/model_definitions_setup.js

```diff
@@ -1,7 +1,7 @@
 /** @odoo-module **/
 
-import { insertModelFields } from '@bus/../tests/helpers/model_definitions_helpers';
+import { insertModelFields } from "@bus/../tests/helpers/model_definitions_helpers";
 
-insertModelFields('res.partner', {
-    out_of_office_date_end: { type: 'date' },
+insertModelFields("res.partner", {
+    out_of_office_date_end: { type: "date", default: () => false },
```

Comment:
IMO: default set to `False` because if not it automatically set `out of office` status for every employee even they are not on leave.

Reply:
Based on above change it should remove the need for this.